### ISSUE TITLE
Rails 5.2.0.rc2 causes ActiveRecord instrumentation exception

### ISSUE THREAD

**Issue by indirect** [scoutapp/scout_apm_ruby#179]
In my Rails 5.2.0.rc2 app, with scout_apm 2.4.9, running any process that enables Scout monitoring results in this log line:

```
[Scout] [04/05/18 08:48:19 +0000 run.6514 (679)] WARN : ActiveRecord instrumentation exception: undefined method `find_with_associations' for module `ActiveRecord::FinderMethods'
```

**Comment by cschneid** [comment]
Yes, we need to check everything against Rails 5.2, which I think is being released imminently.

**Comment by cschneid** [comment]
This was fixed in #182
